dbt_subprojects/dex/macros/models/enrich_dex_liq_with_prices.sql

```diff
@@ -0,0 +1,133 @@
+{% macro enrich_dex_liq_with_prices(
+      base_liquidity = null
+    , tokens_erc20_model = null
+    )
+%}
+
+WITH base_liquidity as (
+    SELECT
+        *
```

Comment:
Minor, but we can list the used columns here:
```suggestion
        blockchain,
        project,
        version,
        block_month,
        block_date,
        block_time,
        block_number,
        id,
        tx_hash,
        evt_index,
        salt,
        token0,
        token1,
        amount0_raw,
        amount1_raw
```